README.md

```diff
@@ -207,7 +207,7 @@ export default class CounterApp {
 
 #### Decorators
 
-The `@connect` decorator lets you create smart components less verbosely:
+The `@connect` decorator lets you create smart components more succinctly:
```

Comment:
I'm not native english speaker, but it's just weird to pronounce it for me. `concisely` is much better :)